Files/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -108,6 +108,16 @@
                                                 <FontIcon FontFamily="{StaticResource CustomGlyph}" Glyph="&#xF107;" />
                                             </MenuFlyoutItem.Icon>
                                         </MenuFlyoutItem>
+                                        <MenuFlyoutItem
+                                            x:Name="PropertiesItem"
+                                            x:Uid="BaseLayoutContextFlyoutPropertiesFolder"
```

Comment:
Can you create a new string called `DrivesWidgetContextFlyoutPropertiesMenuItem`?

Reply:
Yes, I'll make the changes. Just one question(I'm fairly new to this) will it be needed to be created in the resource and string files as well?

Reply:
I have created the new string `DrivesWidgetContextFlyoutPropertiesMenuItem` and I have added a key in the `en-US` resource file as well. Please let me know if anything more needs to be done.

Reply:
@suvrat29 You did it perfectly, thank you!

Reply:
Ah, that's great!